File: Backend/Domain/Model/Automovel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GestaoOfficina.Domain.Model
{
    public class Automovel
    {
        [Key()]
        public int Id { get; set; }
        public string Marca { get; set; }
        public string Placa { get; set; }
        public string Cor { get; set; }
        public string Modelo { get; set; }
        public string Ano { get; set; }
        public string Km { get; set; }
        public int ClientId { get; set; }

        [ForeignKey("Manutence")]
        public int? ManutenceId { get; set; }
        public virtual ICollection<Manutence> Manutences { get; set; }
    }
}

File: FrontEnd/apexv13/src/app/shared/Model/Automovel.ts
import { Identity } from "../crud/Identity";

export class Automovel implements Identity {
  constructor(
    public id: number = 0,
    public marca: string = "",
    public placa: string = "",
    public cor: string = "",
    public modelo: string = "",
    public ano: string = "",
    public km: string = "",
    public clienteId: number = 0
  ) {}

  toJSON(): Object {
    return {
      id: this.id,
      marca: this.marca,
      placa: this.placa,
      cor: this.cor,
      ano: this.ano,
      modelo: this.modelo,
      km: this.km,
      clienteId: this.clienteId
    };
  }
}
